### PR TITLE
Fix personal space name not updating after language change

### PR DESCRIPTION
**Pull request by JammingBen**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8968

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

File: packages/web-runtime/src/pages/account.vue [hunk 1]
```diff
@@ -297,6 +298,14 @@ export default defineComponent({
             value
           }
         })
+        if (unref(personalSpace)) {
+          // update personal space name with new translation
+          store.commit('runtime/spaces/UPDATE_SPACE_FIELD', {
+            id: unref(personalSpace).id,
+            field: 'name',
+            value: $gettext('Personal')
```
Comment (kulmann) [inline packages/web-runtime/src/pages/account.vue:306]: For backends without spaces we don't use `Personal` but `All files`.

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/web-runtime/tests/unit/pages/account.spec.ts [hunk 2]
```diff
@@ -142,17 +142,6 @@ describe('account page', () => {
...
-  describe('computed method "isChangePasswordEnabled"', () => {
-    it('should be true if capability is enabled', () => {
-      const { wrapper } = getWrapper({ capabilities: { spaces: { enabled: true } } })
-      expect(wrapper.vm.isChangePasswordEnabled).toBeTruthy()
-    })
-    it('should be false if capability is not enabled', () => {
-      const { wrapper } = getWrapper()
-      expect(wrapper.vm.isChangePasswordEnabled).toBeFalsy()
-    })
-  })
```
Comment (JammingBen) [inline packages/web-runtime/tests/unit/pages/account.spec.ts:155]: This was just testing the result of the `useCapabilitySpacesEnabled` composable, no need to test that here.

File: packages/web-runtime/src/pages/account.vue [hunk 3]
```diff
@@ -159,18 +156,18 @@ export default defineComponent({
     const disableEmailNotificationsValue = ref<boolean>()
 
     // FIXME: Use graph capability when we have it
-    const isSettingsServiceSupported = useCapabilitySpacesEnabled()
-    const isChangePasswordEnabled = useCapabilitySpacesEnabled()
+    const spacesEnabled = useCapabilitySpacesEnabled()
```
Comment (kulmann) [inline packages/web-runtime/src/pages/account.vue:159]: I liked the aliases of the spaces enabled capability. Reason: It's bad that we use spaces capability for the respective checks. But the aliases limited the defect to a scope where it's easy to understand with the comment above. Using the spaces capability in the template makes it harder to understand that that's not desired...

Comment (JammingBen) [reply packages/web-runtime/src/pages/account.vue:159]: Oh well, okay 😅  Then I'll add it a third time.

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**